pages/gallery.html

```diff
@@ -1,161 +1,47 @@
-          <div class="gallery-item__overlay">
-            <img
-              src="./../assets/images/gallery/expand_icon.svg"
-              class="overlay__image"
-              alt=""
-            />
-          </div>
-        </div>
-        <div class="gallery-item">
-          <div class="gallery-item__images"></div>
-          <div class="gallery-item__overlay">
-            <img
-              src="./../assets/images/gallery/expand_icon.svg"
-              class="overlay__image"
-              alt=""
-            />
-          </div>
-        </div>
-        <div class="gallery-item">
-          <div class="gallery-item__images"></div>
-          <div class="gallery-item__overlay">
-            <img
-              src="./../assets/images/gallery/expand_icon.svg"
-              class="overlay__image"
-              alt=""
-            />
-          </div>
-        </div>
-        <div class="gallery-item">
-          <div class="gallery-item__images"></div>
-          <div class="gallery-item__overlay">
-            <img
-              src="./../assets/images/gallery/expand_icon.svg"
-              class="overlay__image"
-              alt=""
-            />
-          </div>
-        </div>
-        <div class="gallery-item">
-          <div class="gallery-item__images"></div>
-          <div class="gallery-item__overlay">
-            <img
-              src="./../assets/images/gallery/expand_icon.svg"
-              class="overlay__image"
-              alt=""
-            />
-          </div>
-        </div>
-      </div>
-    </section>
-    <footer class="footer">
-      <div class="footer__text">
-        <p>
-          <strong>Hostal donde las paisas.</strong><br>
-          Plan Parejo, Capurgana, 278017 Capurganá, Colombia.
-          <br>
-          <br>WhatsApp: 3116306699
-        </p>
-      </div>
-      <div class="footer__icons-container">
-        <a class="footer__icon" href="https://www.facebook.com/Hostaldondelaspaisas/" target="_blank">
-          <img src="../assets/icons/facebook-logo-button.svg" width="50" height="50">
-        </a>
-        <a class="footer__icon" href="https://www.tripadvisor.co/Hotel_Review-g681277-d12696158-Reviews-Hostal_donde_las_Paisas-Capurgana_Choco_Department.html" target="_blank">
-          <img src="../assets/icons/tripadvisor-logotype.svg" width="50" height="50">
-        </a>
-        <a class="footer__icon" href="https://www.instagram.com/hostaldondelaspaisas/" target="_blank">
-          <img src="../assets/icons/instagram.svg" width="50" height="50">
-        </a>
-      </div>
-      <div class="footer__disclaimer">
-        <span>Web created by </span><a class="footer__footnote" href="https://github.com/rtsc8" target="_blank">RTSC8</a>
-      </div>
-    </footer>
-    <script src="../js/main_nav.js"></script>
-    <script src="../js/gallery.js"></script>
-  </body>
+    </div>
+  </section>
+  <footer>
+    <!-- footer section -->
+  </footer>
```

Comment:
please don't remove the footer element

Reply:
gallery dont have a footer when i do this Pr. I believe with a git pull  i solved that.